lib/senders/templates/new_device_login.html

```diff
@@ -35,6 +35,29 @@ <h1 style="font-family: sans-serif; font-size: 21px; line-height: 29px; font-wei
     {{#if timestamp }}{{ timestamp }}<br/>{{/if}}
 </p>
 
+<!--Button Area-->
+<tr height="50">
+  <td align="center" valign="top">
+    <table border="0" cellpadding="0" cellspacing="0" height="100%" width="100%" id="email-button" style="-webkit-text-size-adjust: 100%; border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt; background-color: #0a84ff; border-radius: 4px; height: 50px; width: 310px !important;">
+      <tr style="page-break-before: always">
+        <td align="center" valign="middle" id="button-content" style="font-family: sans-serif; font-weight: normal; text-align: center; margin: 0; color: #ffffff; font-size: 20px; line-height: 100%;">
```

Comment:
Please also add a test into this file: https://github.com/mozilla/fxa-auth-server/blob/bb2c67747b9ffe2dc398e4bfe7d22438c95cf3a8/test/local/senders/email.js#L206

you need a new `if` statement in there.

If email template is postChangePrimaryEmail or newDeviceLoginEmail.

Something like this (I haven't tested this code):
```
        if (type === 'postChangePrimaryEmail' || type === 'newDeviceLoginEmail' ) {
          it(
            'has manage account link in ' + type,
            function () {
              mailer.mailer.sendMail = function (emailConfig) {
                assert.ok(includes(emailConfig.html, 'Manage devices'))
                assert.ok(includes(emailConfig.text, 'Manage devices'))
              }
              mailer[type](message)
            }
          )
        }
```

Reply:
@ArkadiyVoronov still need to the above to the email.js test file https://github.com/mozilla/fxa-auth-server/blob/bb2c67747b9ffe2dc398e4bfe7d22438c95cf3a8/test/local/senders/email.js#L206